target/linux/ramips/dts/mt7628an_netgear_r6020.dts

```diff
@@ -0,0 +1,173 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "mt7628an.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "netgear,r6020", "mediatek,mt7628an-soc";
+	model = "Netgear R6020";
+
+	aliases {
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+		label-mac-device = &ethernet;
```

Comment:
Earlier, you stated the label shows the 5g MAC address. In that case, this won't work, but you would have to set label_mac in 02_network. Just look for examples with other devices, e.g. cudy,wr1000 ...

Reply:
Ah. I didn't see that.